src/dateutils.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+from dateutil import parser
+import json
+import pytz
+from rfc3339 import rfc3339
+
+def todt(val):
+    "turn almost any formatted datetime string into a UTC datetime object"
+    if val is None:
+        return None
+    dt = val
+    if not isinstance(dt, datetime):
+        dt = parser.parse(val, fuzzy=False)
+    dt.replace(microsecond=0) # not useful, never been useful, will never be useful.
```

Comment:
I read this as the statement is not useful and should, therefore, be removed. Or is it referring to the microsecond value being useless?

Reply:
it's referring to the microsecond value being useless